src/app/authentication/models/otpModel.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import "time"
+
+type OTPValidationRequest struct {
+	Email        string    `gorm:"column:email" json:"email"`
+	OTP          int       `gorm:"column:otp" json:"otp"`
+	CreationTime time.Time `gorm:"column:createdAt" json:"createdAt"`
+}
```

Comment:
use different model for table "users"
create model "user" in src/models/dbModels.go for user table.
remove gorm from here